Packs/CommonScripts/Scripts/CalculateEntropy/CalculateEntropy.yml

```diff
@@ -0,0 +1,35 @@
+args:
+- default: true
+  description: The data for which to calculate entropy.
+  isArray: false
+  name: data
+  required: true
+  secret: false
+- default: false
+  defaultValue: '0'
+  description: The minimum entropy value. Default is 0.
+  isArray: false
+  name: minimum_entropy
+  required: false
+  secret: false
+comment: Calculates the entropy for the given data.
+commonfields:
+  id: CalculateEntropy
+  version: -1
+enabled: false
+name: CalculateEntropy
+outputs:
+- contextPath: EntropyResult.checked_value
+  description: The given value (data).
+  type: String
+- contextPath: EntropyResult.entropy
+  description: The entropy score.
+  type: Number
+script: '-'
+subtype: python3
+system: false
+tags:
+- ''
```

Comment:
This needs to deleted. It appears as empty tag

Reply:
fixed in https://github.com/demisto/content/pull/8294